app/lib/common/widget/AppCommon.dart

```diff
@@ -25,18 +26,14 @@ String getNameFromId(String name) {
   return name.substring(start, end);
 }
 
-String formatedTime(int value) {
-  int h, m;
+String? formatedTime(int value) {
+  final int h = (value.seconds.inHours + 5) % 24;
```

Comment:
why `+ 5`? Does this need some timezone handling?

Reply:
Possibly as based on my location its 5 hours less.